Entity/ExtendedFieldRepositoryTrait.php

```diff
@@ -150,23 +150,35 @@ private function getExtendedFieldValuesMultiple(
         if (empty($extendedFieldList)) {
```

Comment:
$extendedFieldList is not used anymore, so we should drop this requirement and cleanup.